suite-native/atoms/src/Pictogram/CriticalIconSvg.tsx

```diff
@@ -0,0 +1,20 @@
+import Svg, { Path } from 'react-native-svg';
+
+import { CSSColor } from '@trezor/theme';
+
+type CriticalIconSvgProps = {
+    color: CSSColor;
+};
+
+export const CriticalIconSvg = ({ color }: CriticalIconSvgProps) => (
+    <Svg width={40} height={40} fill="none">
```

Comment:
Please create some global const with the size that is reused for all the svgs. So it is easier to change it for all of them at once.

Reply:
The size of SVG icons may vary. Currently, I'd have to create two constants for 40 and 54. In other words, it's just a coincidence that the sizes are the same.

Since we don't do this for other SVG components, I'd leave it as is. I don't wanna create a false expectation that there's some connection between the icons.